app/src/main/res/values-de/strings.xml

```diff
@@ -300,6 +304,9 @@
     <string name="header_live_chats">Live Chats</string>
     <string name="header_unknown">Unbekannt</string>
 
+    <!--Links-->
+    <string name="play_store_link">"https://play.google.com/store/apps/details?id=chat.rocket.android"</string>
```

Comment:
Could you please move this string to the `defaults.xml` file. Don't forget to remove the same string from the others strings.xml file as well. Thank you.